lnrpc/devrpc/dev_server.go

```diff
@@ -227,7 +227,6 @@ func (s *Server) ImportGraph(ctx context.Context,
 	var err error
 	for _, rpcNode := range graph.Nodes {
 		node := &models.Node{
-			HaveNodeAnnouncement: true,
```

Comment:
do we need to apply the workaround here, to not break this? otherwise I think we won't insert the other properties